### DISCUSSION TITLE
Speed up plotting

### DISCUSSION THREAD

**Discussion by mdupilka** [dankelley/oce#2072] [General]
Are there any tips/tricks/suggestions to speed up plotting? I store the plots as a png file.

**Comment by dankelley** [comment]
Not really.  Sometimes oce tries to speed things up e.g.
1. for time-series plots, it decimates to avoid plotting a million points that just go on top of each other
2. it uses base graphics, which are much faster than ggplot graphics

However, some plots just take a while.  If you have a specific plot, perhaps show the code.

@richardsc might have some other ideas.

**Comment by mdupilka** [comment]
Thanks for the reply.  Each plot on its own is pretty fast. It's just that I do a bunch of them. It is for operational purposes, so I am just wondering if it could be speeded up. I do convert data below various thresholds to NAs and skip plotting of those in OCE (I hope).Max

**Comment by dankelley** [comment]
Right, there's probably not much more you can do.  PS. if you reply to github comments over email, you might want to trim the inclusion of the previous material.  Otherwise you'll get as you can see if you read this comment in the link that appears in your email.  My approach is always just to click on that link and to answer in the web form.

Thanks for the question, and best of luck.

Dan.
